example/lib/pages/polygon_custom.dart

```diff
@@ -734,3 +784,58 @@ class PolygonCustomPage extends StatelessWidget {
     );
   }
 }
+
+const _kBaseGroupPaddingHorizontal = 32.0;
+const _kXAxis = 1;
+const _kMinBarSize = 4.0;
+
+/// Changes the position of elements while also updating their width to match
+/// the number of elements in a single band. Useful for bar charts when the
+/// width of the bars can be dynamic.
+@immutable
+class DodgeSizeModifier extends Modifier {
+  @override
+  void modify(AesGroups aesGroups, Map<String, ScaleConv<dynamic, num>> scales,
+      AlgForm form, Offset origin) {
+    final xField = form.first[0];
+    final band = (scales[xField]! as DiscreteScaleConv).band;
+
+    final ratio = 1 / aesGroups.length;
+    final numGroups = aesGroups.length;
+    final groupHorizontalPadding = _kBaseGroupPaddingHorizontal / numGroups;
+    final invertedGroupPaddingHorizontal =
+        coord.invertDistance(groupHorizontalPadding, _kXAxis);
```

Comment:
@entronad I need to get this `coord`, but I can't seem to get it with the current implementation. Do you have any idea how I can convert pixels into the [0, 1] range?